Cli-Shared/Program.cs

```diff
@@ -1113,6 +1215,35 @@ private static bool QueryCredentials(OperationArguments operationArguments)
                     }
                     break;
 
+                case AuthorityType.Bitbucket:
+					{
+                    var bbcAuth = authentication as Bitbucket.Authentication;
+
+                    Task.Run(async () =>
+                    {
+                        if (((operationArguments.Interactivity != Interactivity.Always)
+                             && ((credentials = bbcAuth.GetCredentials(operationArguments.TargetUri, operationArguments.CredUsername)) != null)
+                             && (!operationArguments.ValidateCredentials
+                                 || ((credentials = await bbcAuth.ValidateCredentials(operationArguments.TargetUri, operationArguments.CredUsername, credentials)) != null)))
+                            || ((operationArguments.Interactivity != Interactivity.Never)
+                                && ((credentials = await bbcAuth.InteractiveLogon(operationArguments.TargetUri, operationArguments.CredUsername)) != null)
+                                && (!operationArguments.ValidateCredentials
+                                    || ((credentials = await bbcAuth.ValidateCredentials(operationArguments.TargetUri, operationArguments.CredUsername, credentials)) != null))))
+                        {
+                                Git.Trace.WriteLine($"credentials for '{operationArguments.TargetUri}' found.");
+                                operationArguments.SetCredentials(credentials);
```

Comment:
Hi,

Bitbucket always requires a username, returning 'Personal Access Token, as stored in the credentials manager doesn't work so replace that with the real username

`
                                // Bitbucket relies on a username + secret, so make sure there is a username to return
                                if (operationArguments.CredUsername != null)
                                {
                                    var c2 = new Credential(operationArguments.CredUsername, credentials.Password);
                                    operationArguments.SetCredentials(c2);
                                }
                                else
                                {
                                    operationArguments.SetCredentials(credentials);
                                }
`
(formatting is a bit wacky)

I will look a bit further into this as it maybe the username has changed during the interactive prompts so I just want to be sure my code uses the latest and greatest username, not just the one passed in by Git.